### ISSUE TITLE
Instalação

### ISSUE THREAD

**Issue by Robizidoro** [hiltonbruce/Igreja#2]
Olá, boa tarde.
Consigo instalar usando o Xampp no Win 7  ??

**Comment by hiltonbruce** [comment]
Sim!!!
Mas tem uma biblioteca do PHP q vc tem de instalar manualmente... a DB.php da pear!!!

**Comment by Robizidoro** [comment]
Bom dia.
Sou muito leito nesse tipo de instalação. Tentei de várias formas e não consegui.
Será que poderia meia ajudar ?

**Comment by hiltonbruce** [comment]
Como vc tá tentando... é linux, xamp ... viu a wiki da aplicação?

**Comment by Quirinodsg** [comment]
Bom dia, Hilton!

Fiz a instalação de todos os requisitos e instalei o banco 2022 que estava no arquivo...
Porem não estou achando o arquivo index.html para abrir o sistema.

Outra duvida: http://ip do meu servidor/igreja ou assembleia 
Ou so ip do servidor.

Fiz uma duplicação do index.php para html ele ate carrega a primeira pagina mas com erros de CSS

**Comment by hiltonbruce** [comment]
no caso tem de ser index.php mesmo, provavelmente o interpretador de seu servidor de web não tá reconhecendo a linguagem PHP

**Comment by hiltonbruce** [comment]
Outra duvida: http://ip/ do meu servidor/igreja ou assembleia
Isso vai depender de como e onde vc colocou a pasta

**Comment by hiltonbruce** [comment]
para q eu possa lhe dá informações melhores preciso q me informe o sistema operacional q vc colocou aplicacão e como foi instalado o PHP

Na wiki tem algumas informações q podem lhe ajudar, porém são p Linux Debian e precisam ser adaptados para o q vc usar, mas já dá um norte

**Comment by hiltonbruce** [comment]
Sim, mas vai precisar da classe DB.php q não vai por padrão instalado
Se for realmente instalar me avise q lhe mando o arquivo zipado, eita é de
uma biblioteca da Pegar e no Windows e meio chato de instalar, por isso
mando o zip q é mais rápido

Em sáb, 21 de ago de 2021 15:19, Robizidoro ***@***.***>
escreveu:

**Comment by hiltonbruce** [comment]
O DB.php é de uma biblioteca da Pear, o corretor alterou o nome!!!

Em qui, 8 de set de 2022 20:25, Hilton Bruce ***@***.***>
escreveu:

**Comment by marconessb** [comment]
Boa noite, teria como me enviar o arquivo DB.php não veio na pasta, estou tentando rodar no xampp esta dando erro...

Qual a versão do php?

**Comment by hiltonbruce** [comment]
deve ser extraído na pasta php p que se tenha acesso a classe e seus
arquivos....
Joseilton C Bruce


Em qua., 14 de set. de 2022 às 19:07, marconessb ***@***.***>
escreveu:

**Comment by Emanuelkeys** [comment]
Boa tarde, os senhores tem alguma ferramenta para organizar ministério de louvor? 
Como organizar repertório por exemplo?

**Comment by hiltonbruce** [comment]
Infelizmente não... Não houve essa demanda aqui e por isso nem pensei em
fazer algo do tipo!!!
Joseilton C Bruce


Em ter., 30 de jan. de 2024 às 14:58, Emanuelkeys ***@***.***>
escreveu:
